alembic/versions/0210dc9b2f44_migration_message.py

```diff
@@ -0,0 +1,28 @@
+"""Add a resolved column to the responses table
+
```

Comment:
Give this file a name relevant to what it does

Reply:
okay

Reply:
This has been worked on @Bonifase